test/util/tokens.js

```diff
@@ -135,6 +154,13 @@ define(function() {
                                'JOEhR6vs11qVmIgbwZ4--MqUIRU3WoFEsr0muLl039QrUa1' +
                                'EQ9-Ua9rPOMaO0pFC6h2lfB_HfzGifXATKsN-wLdxk6cgA';
 
+  tokens.standardAccessTokenParsed = {
+    accessToken: tokens.standardAccessToken,
+    expiresAt: 1449703529, // assuming time = 1449699929
```

Comment:
nope, that was https://github.com/okta/okta-auth-js/pull/35/files#diff-dbfa077c685241b22c18a8a43ee006fbR72

I just wanted to make sure readers knew the default on accessTokens assumes a specific time, because the oauth response for access_tokens returns an expires_in, not expires_at.